src/Aspire.Hosting/ApplicationModel/ParameterDefault.cs

```diff
@@ -27,6 +27,39 @@ public abstract class ParameterDefault
 /// <summary>
 /// Represents that a default value should be generated.
 /// </summary>
+/// <remarks>
+/// The recommended minimum bits of entropy for a generated password is 128 bits.
```

Comment:
I'm pretty sure Visual Studio and other tools don't care about whitespace. This content in a tooltip will be one blob of text.

If you want paragraphs, which IMO we should have, then you must add <para> tags.

Reply:
Current tooltip:

![image](https://github.com/dotnet/aspire/assets/303201/62ec86ff-cc43-45a8-8fb8-d35d395463a0)

Reply:
The tooltip now looks like:
![image](https://github.com/dotnet/aspire/assets/8291187/4204d5db-20f4-4e74-b258-354c0918e1d3)